gatsby-config.js

```diff
@@ -108,5 +130,17 @@ module.exports = {
       },
     },
     'gatsby-plugin-remove-serviceworker',
+    {
+      resolve: `gatsby-source-meetup`,
+      options: {
+        apiKey: process.env.GATSBY_MEETUP_API_KEY,
```

Comment:
Can you send me privately the key to use?

Reply:
Oops... I don't even use this plugin anymore, it's an obsolete code remnant, I'll remove it as soon as I can !